lib/src/utf8.dart

```diff
@@ -22,6 +22,7 @@ extension Utf8Pointer on Pointer<Utf8> {
   /// The UTF-8 code units of the strings are the non-zero code units up to the
   /// first zero code unit.
   int get length {
+    _ensureNotNullptr();
     final Pointer<Uint8> array = cast<Uint8>();
```

Comment:
Possibly stupid, but remind me why this cast is necessary when `this` has static type `Pointer<Uint8>`?
Is it to make sure it's not a subtype?
(Then document it, so I won't have to ask again 😊 )

Reply:
`this` has static type `Pointer<Utf8>`, `Utf8` != `Int8`.

Reply:
I'll rename `array` to `codeUnits`.

Reply:
Also, remind me not to be blind 🙈 .